desktop_version/src/Graphics.cpp

```diff
@@ -2,6 +2,7 @@
 #include "Graphics.h"
 
 #include <SDL.h>
+#include <algorithm>
```

Comment:
Includes need to be in alphabetical order.
```suggestion
#include <algorithm>
#include <SDL.h>
```